app/features/config/SelectWowDir.tsx

```diff
@@ -2,7 +2,7 @@ import React, { useCallback, useState, SyntheticEvent } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import { Button, Snackbar, CircularProgress } from '@material-ui/core';
 import { Alert } from '@material-ui/lab';
-import { remote } from 'electron';
+import { app, remote } from 'electron';
```

Comment:
Does this work? According to docs `app` is only available to the main-thread. So I would think you would need to do `remote.app.getPath`. Or you could do it like you did before, with `os.homedir()`

Reply:
A bit hard for me to test at the moment as the select wow path snackbar stopped showing up (even on your master branch). I'll just revert it to os.homedir() with proper import. The reason I tried using app.getPath was I saw a comment on it being in the electron api. So I figured that would be more proper.